custom/icds_reports/ucr/reports/mpr_2bi_preg_delivery_death_list.json

```diff
@@ -109,61 +109,26 @@
         "format": "default",
         "transform": {
           "type": "translation",
+          "mobile_or_web": "mobile",
```

Comment:
@sheelio FYI, I'm reformatting the ICDS translation transforms to be in line with the new spec.  This shouldn't have any functional differences.